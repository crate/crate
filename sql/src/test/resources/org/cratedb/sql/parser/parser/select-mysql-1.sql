SELECT * FROM t1 USE INDEX (idx_1, idx_2), t2