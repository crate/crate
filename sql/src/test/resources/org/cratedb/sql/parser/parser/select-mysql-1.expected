org.cratedb.sql.parser.parser.CursorNode@33aae94f
name: null
updateMode: UNSPECIFIED
statementType: SELECT
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@19bb5c09
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@5b976011

		[0]:		
		org.cratedb.sql.parser.parser.AllResultColumn@2f242b11
		tableName: null
		exposedName: null
		name: null
		tableName: null
		isDefaultColumn: false
		type: null
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@6f878144

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@2937d090
		tableName: t1
		updateOrDelete: null
		null
		correlation Name: null
		null
		indexHints: 			
			org.cratedb.sql.parser.parser.IndexHintList@f5db314

			[0]:			
			org.cratedb.sql.parser.parser.IndexHintNode@5097eded
			hintType: USE
			hintScope: null
			indexes: [idx_1, idx_2]
		[1]:		
		org.cratedb.sql.parser.parser.FromBaseTable@26858bf1
		tableName: t2
		updateOrDelete: null
		null
		correlation Name: null
		null